pkg/api/v1beta2/dynakube/validation/validation.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"errors"
 
-	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
should be v1beta2

Reply:
not only that but in the config the name of the webhook should be different: https://github.com/Dynatrace/dynatrace-operator/blob/99b4b16a4434dc00a47fecb6c434ffec15c084da/config/helm/chart/default/templates/Common/webhook/validatingwebhookconfiguration.yaml#L63

otherwise you will overwrite it

Reply:
example:
`edgeconnect.webhook.dynatrace.com` is the name of the edgeconnect webhook

Reply:
we should call it something like `version.cr-name.webhook.dynatrace.com`